pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/ddp/kitrequest/KitRequestDao.java

```diff
@@ -32,7 +32,7 @@
 
     public static final String SQL_GET_KIT_REQUEST_ID = "SELECT  ddp_kit_request_id  FROM  ddp_kit_request";
 
-    public static final String BY_BSP_COLLABORATOR_PARTICIPANT_ID = " WHERE bsp_collaborator_participant_id = ?";
+    public static final String BY_BSP_COLLABORATOR_SAMPLE_ID = " WHERE bsp_collaborator_sample_id = ?";
```

Comment:
why are we changing the search criteria here?

Reply:
because for all the kits of a participant,  their participant ids the same but the sample id gets a `_counter` that will show which one we want

Reply:
ok, makes sense, but then I think we should rename some things here: `public String getKitRequestIdByBSPParticipantId(String bspParticipantId)` to 
public String getKitRequestIdByBspSampleId(String bspSampleId)`

Reply:
Done
Thank you